sos_trades_core/tests/l0_test_02_sos_discipline.py

```diff
@@ -350,3 +354,39 @@ def test_10_check_overwrite_of_default_values(self):
         out_dict = disc8.get_sosdisc_outputs()
         ref_out = {'indicator': 200.0, 'y': 120.0}
         self.assertDictEqual(ref_out, out_dict, 'error in input dict')
+
+    def test_11_check_distortion_success(self):
+        ''' Test successful DiscDistortion run '''
+        study_name = 'TestDistortionSuccess'
+        disc_name = 'DiscDistortion'
+        limit = 0.7
+
+        # create execution engine
+        self.ee = ExecutionEngine(study_name)
+        self.ee.ns_manager.add_ns('ns_distortion', study_name)
```

Comment:
how do we use namespaces?

Reply:
namespace are used to couple variables:

if the output of your discipline is 'wave_limit' it will be only visible by your discipline (not so useful)

if 'wave_limit' is used by another discipline as input, then you can use namespace in the DESC_OUT variable description and then use the same namespace for your new discipline . The execution engine will then couple the variable in the corresponding namespace

Reply:
That's interesting, I will try coupling by execution engine

Reply:
All right, I was able to write a 2nd discipline, and couple it with this one, pipeline-style.

As my next step, I will try to couple real disciplines from witness-core and witness-energy